src/modules/channel9.js

```diff
@@ -0,0 +1,17 @@
+export default {
+  name: "Channel 9",
+  domains: ["channel9.msdn.com"],
+  getPatterns: function() {
+    return [
+      /channel9\.msdn\.com\/[\w-]/i
```

Comment:
This matches too many URLs. Try visiting https://channel9.msdn.com/Shows/DevOps-Lab/ARM-Series-12-Azure-DevOps-With-ARM-Templates, the entire page is broken.